cdap-docs/developers-manual/source/pipelines/developing-plugins/presentation-plugins.rst

```diff
@@ -33,7 +39,9 @@ The widget JSON is composed of:
 
 - a map of :ref:`metadata <plugins-presentation-metadata>`
 - a list of property :ref:`configuration groups <plugins-presentation-configuration-groups>`
+- a map of :ref:`inputs  (input properties) <plugins-presentation-inputs>`
 - a list of :ref:`outputs (output properties) <plugins-presentation-outputs>`
```

Comment:
In the case of output it is the property that holds the schema of the data that goes out.